### PR TITLE
unbind config.stmp.auth.user and config.smtp.auth.pass from config.smtp.secure flag

### PR DESCRIPTION
**Pull request by AlvaroVega**
fix for issue https://github.com/telefonicaid/perseo-fe/issues/514

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Fix config smtp auth user and password: unbind them from config smtp secure flag
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: No related issue?

**Comment by fgalan** [comment]
Is this PR a "clear version" of existinng PR https://github.com/telefonicaid/perseo-fe/pull/454 ?

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: fixed in https://github.com/telefonicaid/perseo-fe/pull/516/commits/317b1b3b9558956013609db45233d3f77ed1fe7e

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by AlvaroVega** [comment]
- [ ] Check this execution:

time=2021-04-08T22:18:31.932Z | lvl=DEBUG | corr=n/a | trans=n/a | op=start | comp=perseo-fe | msg=Using config:

{
"logLevel": "DEBUG",
"endpoint": {
"host": "localhost",
"port": 9090,
"rulesPath": "/rules",
"actionsPath": "/actions/do",
"noticesPath": "/notices",
"vrPath": "/m2m/vrules",
"checkPath": "/check",
"versionPath": "/version",
"logPath": "/admin/log",
"metricsPath": "/admin/metrics"
},
"isMaster": true,
"slaveDelay": 60000,
"mongo": {
"url": "mongodb://10.0.0.19:27017,10.0.0.20:27017,10.0.0.21:27017/cep?replicaSet=cb_rs0"
},
"orionDb": {
"url": "mongodb://10.0.0.19:27017,10.0.0.20:27017,10.0.0.21:27017/orion?replicaSet=cb_rs0",
"collection": "entities",
"prefix": "orion",
"batchSize": 500
},
"perseoCore": {
"rulesURL": "http://localhost:8080/perseo-core/rules",
"noticesURL": "http://localhost:8080/perseo-core/events",
"interval": 60000
},
"nextCore": {
"rulesURL": "http://10.0.0.34:8080/perseo-core/rules",
"noticesURL": "http://10.0.0.34:8080/perseo-core/events"
},
"smtp": {
"port": 9999,
"host": "10.0.0.24",
"secure": false,
"tls": {
"rejectUnauthorized": false
},
"name": "int2-",
"auth": {}
},
"sms": {
"URL": "http://10.0.0.24:9998/send/sms",
"API_KEY": "80",
"API_SECRET": "22",
"from": "tel:nophone1;phone-context=+34"
},
"smpp": {
"host": "10.0.0.24",
"port": "9997",
"systemid": "user",
"password": "pass",
"from": "tel:nophone1;phone-context=+34",
"enabled": false
},
"orion": {
"URL": "http://10.0.0.2:1026"
},
"pep": {
"URL": "http://10.0.0.2:10026"
},
"authentication": {
"host": "keystone",
"port": "5001",
"user": "user",
"password": "password"
},
"collections": {
"rules": "rules",
"executions": "executions"
},
"executionsTTL": 86400,
"DEFAULT_SUBSERVICE": "/",
"DEFAULT_SERVICE": "unknownt",
"checkDB": {
"delay": 5000,
"reconnectTries": 1000,
"reconnectInterval": 5000,
"bufferMaxEntries": 5
},
"restBase": null,
"castTypes": false
}

**Pull request merged**